### ISSUE TITLE
Is postman compatible with this api?

### ISSUE THREAD

**Issue by darias08** [andshrew/PlayStation-Trophies#16]
Hello,

I am trying to use Postman from Playstation-trophies api, but I am getting an error message saying: 

{
    "error": {
        "referenceId": "e435efb4-70fd-11ed-99c3-01c61dc733aa",
        "code": 2241025,
        "message": "Invalid token"
    }
}

Here's a screenshot of what I am setting up with, but let me know if postman is compatible with this api. Thank you!(:

![psn](https://user-images.githubusercontent.com/58616895/204922017-3a2b4639-af0c-41bc-aa01-2e2f5dd78064.PNG)

**Comment by rafaelfgyn** [comment]
Hello darias08.

You're passing the wrong way. Go in authorization tab and choose type "Bearer token" then passa the ONLY the token in the input without double quotes.

Good luck!

**Comment by darias08** [comment]
Hello!

So I did place the type as Bearer Token and within the token I put myNpsso, but it's still giving me the same type of error message saying, "Invalid token".

**Comment by darias08** [comment]
Also quick question, is it possible I can pull the game cover url of a title from this api?

**Comment by rafaelfgyn** [comment]
oh no man, I didn't notice that you're passing your npsso key. You shouldn't. 
you use the npsso to generate a valid token THEN put in the authorization tab

Answering your last question, yes! it's possible.

you can have a better idea of what is possible accessing this page made by myself. psntrophies.com.br

**Comment by darias08** [comment]
Hi, 

So how exactly do I generate a token from using my npsso? 

Because I am reading through this post from 
https://andshrew.github.io/PlayStation-Trophies/#/APIv2?id=additional-endpoints 

to get the Auth token, but that just uses myNppso to grant access. It doesn't generate a token for me. So how can I get this token?

**Comment by rafaelfgyn** [comment]
about that code, Instead of Write-Host "Authentication Token successfully granted" try "Write-Host $token" to see the content of that token.

**Comment by darias08** [comment]
So I replaced "Authentication Token successfully granted" to Write-Host $token" but it gives me an output of 

Write-Host System.Security.SecureString
Did I replace it correctly? 


![psn2](https://user-images.githubusercontent.com/58616895/204936619-38296eec-a7e0-4c68-b4ee-d03d2ec92bd0.PNG)

**Comment by rafaelfgyn** [comment]
sorry that was my mistake

this whole line should be => 
`
Write-Host $result
`
dont need to write Write-Host twice and no need of double quotes too

**Comment by darias08** [comment]
Okay I got it! Thank you! :D

**Comment by darias08** [comment]
Okay, but back to the getting the image question. So you said I would have to use this link: psntrophies.com.br to find what is the endpoint for a game cover image id, right?

**Comment by darias08** [comment]
Because under the api the only image I can see to call is the icon which isn't the game cover. 

![PSNAPI](https://user-images.githubusercontent.com/58616895/204947649-43eb4ddd-85dd-4df6-9a9b-58330415b613.PNG)

**Comment by rafaelfgyn** [comment]
> Okay, but back to the getting the game image question. So you said I would have to use this link: psntrophies.com.br to find what is the endpoint for a game cover image id, right?

No, this site is only for you to have a idea of what is possible because it uses many of the resources available

**Comment by darias08** [comment]
Is there a github repo for that site?

**Comment by rafaelfgyn** [comment]
just to be sure we're talking about the same thing, what do you mean by "game cover"?

**Comment by darias08** [comment]
Like this is what I am talking about. 

https://psntrophies.com.br:4000/Games/NPWR22392_00/cover.png

**Comment by rafaelfgyn** [comment]
> Because under the api the only image I can see to call is the icon which isn't the game cover.
> 
> ![PSNAPI](https://user-images.githubusercontent.com/58616895/204947649-43eb4ddd-85dd-4df6-9a9b-58330415b613.PNG)

if I"m not mistaken this "trophyTitleIconUrl" should give you the cover, is It not?

**Comment by darias08** [comment]
Yeah, but that's just the title of the game not really a cover art. Is there any possibility I can get the cover art instead? Like the one I showed you the link.

**Comment by darias08** [comment]
I guess what I am saying is the box art cover if there is one.

**Comment by rafaelfgyn** [comment]
ah ok got it. you can't have cover arts  in this api as far as I can tell.

**Comment by darias08** [comment]
Oh, bummer. Well anyway I guess I will close this issue since I got everything else resolved. Thank you for your helping me with this api! :D
